### DISCUSSION TITLE
22. SSL과 TLS는 어떤 차이가 있나요?

### DISCUSSION THREAD

**Discussion by eun-byeol** [jewelry-box/cs-interview#24] [General]
SSL과 TLS는 어떤 차이가 있나요?

**Comment by linirini** [comment]
SSL과 TLS는 데이터를 안전하게 전송하기 위해 사용되는 암호화 프로토콜이다.
TLS는 SSL의 **업데이터된 버전**이다. SSL은 넷스케이프에 의해서 발명되어 점차 폭넓게 사용되다가 표준화 기구인 IETF의 관리로 바뀌면서 TLS라는 이름으로 변경된 것이다. 
현재는 여전히 SSL 인증서라는 용어로 불리지만, 실제로는 TLS 프로토콜을 의미한다.
